examples/terragrunt/README.md

```diff
@@ -0,0 +1,43 @@
+# Terragrunt
+
+[//]: <> (BEGIN EXAMPLE)
+```yml
+name: Terragrunt
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  terragrunt:
+    name: Terragrunt
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Install Terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_wrapper: false # This is required so that Terraform binary outputs valid JSON
```

Comment:
Should we report this to them? is https://github.com/hashicorp/setup-terraform/issues/42 the issue?

Reply:
Yeah, that's the same issue.